### DISCUSSION TITLE
error: websocket.js:88 WebSocket connection to 'wss://myDomain/socket.io/?EIO=4&transport=websocket&sid=bgg8FwFMinIGqssUAAAE' failed: Error during WebSocket handshake: Unexpected response code: 400

### DISCUSSION THREAD

**Discussion by krzysztofalicki** [miguelgrinberg/Flask-SocketIO#1764] [Q&A]
hey @miguelgrinberg , hey All

i have a question, because of the error in the topic:
websocket.js:88 WebSocket connection to 'wss://myDomain/socket.io/?EIO=4&transport=websocket&sid=bgg8FwFMinIGqssUAAAE' failed: Error during WebSocket handshake: Unexpected response code: 400

I am using:
flask,
socketio,
nginx,
wsgi,
gunicorn

tried with Your(@miguelgrinberg) advice about CORS but doesnt work :(

how we start the app:

flask_srv.py:
app = Flask(name,
static_url_path='',
static_folder=static_dir,
template_folder=templates_dir)
CORS(app)
socketio = SocketIO()
socketio.cors_allowed_origins = "*"
socketio.logger = True
socketio.init_app(app)

socketio.js:
var socket = io.connect(null, {port: 7000, transports: ['websocket'], origins: ':'});

Waiting for some advices :)
Thank you

**Comment by miguelgrinberg** [comment]
Look at the output from the server. Maybe there are some clues there.
